src/core/settings/production.py

```diff
@@ -74,3 +74,4 @@
 ANYMAIL = {
     'SENDINBLUE_API_KEY': SIB_API_KEY,
 }
+SIB_WELCOME_EMAIL_TEMPLATE_ID = env.int('SIB_WELCOME_EMAIL_TEMPLATE_ID', 41)
```

Comment:
```suggestion
SIB_WELCOME_EMAIL_TEMPLATE_ID = env.int('SIB_WELCOME_EMAIL_TEMPLATE_ID')
```
je ferai comme `ANALYTICS_SITEID`